pkg/util/mock/context.go

```diff
@@ -450,6 +450,11 @@ func (c *Context) InSandBoxMode() bool {
 	return c.inSandBoxMode
 }
 
+// SetInfoSchema implements the sessionctx.Context interface.
```

Comment:
Setting InfoSchema manually seems dangerous, please comment this is only for test, do not use it in TiDB.

Reply:
This package is only used in the test. it is a mock context.

Reply:
Done